include/swift/SILOptimizer/OptimizerBridging.h

```diff
@@ -0,0 +1,37 @@
+//===--- OptimizerBridging.h - header for the OptimizerBridging module ----===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_SILOPTIMIZER_OPTIMIZERBRIDGING_H
+#define SWIFT_SILOPTIMIZER_OPTIMIZERBRIDGING_H
+
+#include "../SIL/SILBridging.h"
```

Comment:
We generally do not do relative paths like this in the include dir. Can you do the full "swift/SIL/SILBridging.h"?

Reply:
In this case it's needed for the libswift build.